tests/manage/z_cluster/upgrade/test_check_mon_pdb_post_upgrade.py

```diff
@@ -4,22 +4,31 @@
 from semantic_version import Version
 
 from ocs_ci.framework import config
-from ocs_ci.framework.testlib import post_ocs_upgrade, ManageTest, skipif_external_mode
+from ocs_ci.framework.testlib import (
+    post_ocs_upgrade,
+    ManageTest,
+    skipif_external_mode,
+    post_ocp_upgrade,
+)
+from ocs_ci.ocs.ocp import get_ocs_version
 from ocs_ci.ocs.cluster import CephCluster
 from ocs_ci.helpers.helpers import get_mon_pdb
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import validate_existence_of_blocking_pdb
+from ocs_ci.ocs.resources.pod import get_all_pods
 
 log = logging.getLogger(__name__)
 
 
-@post_ocs_upgrade
-@skipif_external_mode
-@pytest.mark.polarion_id("OCS-2449")
 class TestToCheckMonPDBPostUpgrade(ManageTest):
```

Comment:
Now the test file "test_check_mon_pdb_post_upgrade.py" consists of both mon and osd pdb test functions. Please also change the test file name accordingly or make it generic something like test_pdb_check_post_upgrade.py 
Same for the class function also.

Reply:
DOne